crates/red_knot_python_semantic/resources/mdtest/suppressions/knot_ignore.md

```diff
@@ -77,7 +77,8 @@ def test(a: f"f-string type annotation", b: b"byte-string-type-annotation"): ...
 ```py
 # error: [invalid-syntax]
 # error: [unused-ignore-comment]
-def test(  # knot: ignore
```

Comment:
The previous syntax error here is kind of brittle for this test, as it moves to the next line *if there is a trailing newline in the file*. I made a minor modification here that unambiguously places the syntax error in this line.